packages/version-select/docs.mdx

```diff
@@ -4,21 +4,9 @@ componentName: VersionSelect
 
 <p>{packageJson.description}</p>
 
-_Note:_ `VersionSelect` uses `next/router`. Interacting with it in **this playground**
-will navigate you to non-existant pages.
-
-<LiveComponent>{`<VersionSelect
-  versions={[
-      { name: "latest", label: "latest" },
-      { name: "v0.5.1", label: "v0.5.1" },
-      { name: "v0.4.0", label: "v0.4.0" },
-  ]}
-/>`}</LiveComponent>
-
-With Next.js router debugger
-
```

Comment:
Removing this since `VersionSelect` without a `NextRouterDebugger` wrapper is not usable in a playground setting